CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Guia de Como Contribuir
+
+## Para os novos contribuidores
+
+1. Faça um fork do projeto.
+2. Crie uma nova branch com as suas alterações: 
+3. `git checkout -b nome-pagina-nome-secao` (altere nome-pagina-nome-secao de acordo com o 
+nome da página ou seção que você está traduzindo).
+4. Salve as alterações e crie uma mensagem de commit contando o que você fez:
+`git commit -m "doc: Add Example page"`.
+5. Envie as suas alterações: `git push origin nome-pagina-nome-secao`.
+6. Verifique se a sua branch está atualizada com a branch `master` e submeta seu pull request!
```

Comment:
I suggest to add command examples for step 6 as you did for 3, 4 and 5.

Reply:
@lucastfa-toptal added now